sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -381,7 +383,7 @@ def peek(self, message_count=1, sequence_number=None):
                 :caption: Look at pending messages in the queue.
 
         """
-        self._can_run()
+        self._check_session()
```

Comment:
It feels wrong that this is in non-session receiver, given that we split the two.  Should either be on _can_run with an overload tailored for each circumstance, or should not need to call this.

Reply:
(same feeling about calling `check_session` in a non-session receiver)

Reply:
Removed it from non-session receiver

Reply:
Changed to _check_live